proto/tendermint/abci/types/types.proto

```diff
@@ -246,11 +246,12 @@ message ResponseOfferSnapshot {
   Result result = 1;
 
   enum Result {
-    ACCEPT        = 0;  // Snapshot accepted, apply chunks
-    ABORT         = 1;  // Abort all snapshot restoration
-    REJECT        = 2;  // Reject this specific snapshot, try others
-    REJECT_FORMAT = 3;  // Reject all snapshots of this format, try others
-    REJECT_SENDER = 4;  // Reject all snapshots from the sender(s), try others
+    UNKNOWN       = 0;  // Unknown result, abort all snapshot restoration
```

Comment:
why do we need UNKNOWN items again?

Reply:
If the field isn't specified, or it contains a value not in the enum, it defaults to 0 which is `ACCEPT`. This means if anyone returns empty noop responses, i.e. accept, state sync will believe the snapshots have been restored when they haven't - things will go poorly from there.

Both Protobuf docs and linters recommend always having an unknown zero value to avoid these sorts of problems.